nebula/ui/components/VPNMainImage.qml

```diff
@@ -56,8 +55,8 @@ Rectangle {
 
         },
         State {
-            name: "stateDisconnecting"
```

Comment:
this seems to be an mistake :)

Reply:
definitely a mistake. Will fix this up and push to the gg branch.